pkgs/desktops/gnome-3/extensions/buildGnomeExtension.nix

```
@@ -0,0 +1,42 @@
+{ pkgs, lib, stdenv, fetchurl, gnome3, unzip }:
+
+let
+  buildGnomeExtension = {
+    # Every gnome extension has a UUID. It's the name of the extension folder once unpacked
+    # and can always be found in the metadata.json of every extension.
+    uuid,
+    name,
+    pname,
+    description,
+    # extensions.gnome.org extension URL
+    link,
+    version,
+    sha256,
```

```suggestion
    , uuid
    , name
    , pname
    , description
    # extensions.gnome.org extension URL
    , link
    , version
    , sha256
```

normally it is done like this.

---

I thought it was a work-around for not having trailing commas in nix, so that the diff is cleaner when adding lines at the end. Is this still a thing, now that we have trailing commas? Or are we simply keeping up with that style for consistency?